### ISSUE TITLE
[Documentation] Missing / Wrong BNF syntax for Update Sheet / Doc

### ISSUE THREAD

**Issue by jay-eleven** [taers232c/GAMADV-XTD3#400]
Hi Ross!

I was just reviewing your recent addition in 6.71.07 [Added command to upload changes to Google Docs](https://github.com/taers232c/GAMADV-XTD3/wiki/Users-Drive-Files-Manage#upload-changes-to-google-documents) and found some inconsistencies.

I already updated the Wiki with some minor typos but I think there's still stuff missing.

[Upload changes to Google Documents](https://github.com/taers232c/GAMADV-XTD3/wiki/Users-Drive-Files-Manage#upload-changes-to-google-documents)

```
gam <UserTypeEntity> update document <DriveFileEntity>
        ((json [charset <Charset>] <SpreadsheetJSONUpdateRequest>) |
         (json file <FileName> [charset <Charset>]))
        [formatjson]
```

`<SpreadsheetJSONUpdateRequest>` is not defined on this page. So I browsed to [Users - Spreadsheets](https://github.com/taers232c/GAMADV-XTD3/wiki/Users-Spreadsheets#users---spreadsheets) and, sure enough, I found the [definition](https://github.com/taers232c/GAMADV-XTD3/wiki/Users-Spreadsheets#definitions):

```
<SpreadsheetJSONUpdateRequest> ::=
        '{"requests": [{<Request>}(,{<Request>})], "includeSpreadsheetInResponse": true|false,
          "responseRanges": ["<SpreadsheetRange>"(,"<SpreadsheetRange>")],
          "responseIncludeGridData": true|false}
```

This raised a new question as I couldn't find the definition of `<Request>`.

Furthermore, I don't think `<SpreadsheetJSONUpdateRequest>` is suitable for `gam update document` (Docs) as it contains Sheets specific options which don't apply to Docs: `includeSpreadsheetInResponse`, `responseRanges` and `responseIncludeGridData`, so a new `<DocumentJSONUpdateRequest>` is needed. 🤓

So, to sum up, I think the following symbols are needed:

1. Create a new `<DocumentJSONUpdateRequest>` for `gam update document`
2. Define `<Request>` (or `<DocumentRequest>`) for `<DocumentJSONUpdateRequest>`
3. Define `<Request>` (or `<SpreadsheetRequest>`) for `<SpreadsheetJSONUpdateRequest>`

As always, if I can be of any assistance, just let me know. Thanks for your hard work!

**Comment by taers232c** [comment]
Thanks. I made some updates.
----
Ross Scroggs
***@***.***
